Dockerfile

```diff
@@ -0,0 +1,42 @@
+# created by Naga Ravi Chaitanya Elluri
+# Edited by Naga Ravi Chaitanya Elluri
+
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update
+RUN apt-get -y install make python lsb-release unzip wget vim git curl
```

Comment:
These two lines must be collapsed into a one-liner:
```Dockerfile
RUN apt-get update \
    && apt-get -y install make python lsb-release unzip wget vim git curl
```
The reason for this is that docker caches each `RUN` (or equivalent command) as a separate layer, and having a cached `apt-get update` can only cause problems if it gets stale. We've seen this cause CI builds to fail before, so the rule of thumb is `apt-get update && apt-get install` must be an "atomic" operation.